### DISCUSSION TITLE
Ajout de leagues

### DISCUSSION THREAD

**Discussion by Pietagorh** [Crownicles/Crownicles#581] [Idées acceptées / déjà implémentées]
Pouvoir obtenir des rangs, comme chevalier par exemple. Ou encore aventurier.
Que la réputation ( points ) puisse avoir un impact sur l'aventure.

**Comment by BastLast** [comment]
je déplace dans accepté parce que techniquement ça arrive avec la maj des fight meme si c'est pas exactement ce qui est décris. Go ne pas faire de suggestions aussi vague la prochaine fois tho
